src/components/Sidebar/SidebarItem.tsx

```diff
@@ -6,17 +6,31 @@ import SidebarIcon from './SidebarIcon';
 
 type IconLabel = 'Home' | 'Commits' | 'Contributors' | 'Reviews' | 'Quality';
 
-export default function SidebarItem({ label }: { label: IconLabel }) {
+export default function SidebarItem({
+  label,
+  disabled = false,
+}: {
+  label: IconLabel;
+  disabled?: boolean;
+}) {
   const pathname = usePathname();
   const isActive =
     pathname === `/${label.toLowerCase()}` ||
     (label === 'Home' && pathname === '/');
 
+  const baseHref =
+    label.toLowerCase() === 'home' ? '/' : `/${label.toLowerCase()}`; // 'href' for sidebar menu links
+
   return (
     <li>
       <a
-        className={`${cn(isActive && 'menu-active')} flex items-center justify-between`}
-        href={label.toLowerCase() === 'home' ? '/' : `/${label.toLowerCase()}`}
+        href={disabled ? undefined : baseHref} // If not disabled, 'baseHref' returns url linU for menu item
```

Comment:
Please remove the comments on this line and the line with aria-disabled. They are making the code more difficult to understand, which is not what we want.